src/docs/sdk/event-payloads/properties/measurements.mdx

```diff
@@ -0,0 +1,18 @@
+`measurements`
+
+: _Optional_. An object containing standard/custom measurements with keys signifying the name of the measurement.
+
+Standard measurement keys currently supported are from the set  `"fp" | "lcp" | "fid" | "ttfb" | "ttfb.requesttime"`.
```

Comment:
I took these from [here](https://github.com/getsentry/sentry/blob/6f31e2fc489a823c9395bc23fcd5389b3dff509c/src/sentry/discover/arithmetic.py#L156-L161) and also from the various setters in [`sentry-javascript/metrics`](https://github.com/getsentry/sentry-javascript/blob/f715e8754c11797120989e52e2342c89b98b5ea1/packages/tracing/src/browser/metrics.ts#L235). Feel free to add/update this list.

Reply:
Hmm interesting that the mobile vitals are not in the Sentry allow list. So you can't do discover arithmetic on them?

Reply:
there is another list [here](https://github.com/getsentry/sentry/blob/6f31e2fc489a823c9395bc23fcd5389b3dff509c/src/sentry/search/events/constants.py#L124-L134) which also has `cls`.

Reply:
Hmm perhaps @wmak can help figure this out here

Reply:
Hmm not being able to do arithmetic on vitals is definitely an oversight on my part (ie. we didn't update this allowlist when working on it)

But i believe the standard measurement list should be everything we prefix measurements on in the UI? ie. we should include the ones [here](https://github.com/getsentry/sentry/blob/a8c960a933d2ded5225841573d8fc426a482ca9c/static/app/utils/discover/fields.tsx#L654-L676) that aren't rates or percentages (those are computed)